### PR TITLE
Clean manifest

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
## Purpose / Description
Initially, I was trying to fix a bug by changing some data in manifest. It didn't work. I'm doing it another way, this will be shown in a future PR.

While changing the manifest, I found that it can be syntactically improved. Some indentation were missing. Some > and /> should be on their own line I believe (to be consistent with } being on its own line, and to avoid having useless diff in future commit if parameters are added to tags latter.

And I moved the most important (i.e. unique) information as first paramaters of tags. I.e. name and label.

## How Has This Been Tested?

Starting anki, and seing that I can still use it normally.

I strongly suggest reviewing each commit individually. The whole diff is hard to grasp, but each commit should be quite simple to understand, and to check that everything still works as intended.

## Learning (optional, can help others)

That adding `android:launchMode="singleTop"` in an activity does not actually prevent this activity to be opened twice, both being at the top. Quite disappointing. 

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code

### CODE REVIEW

**Comment by Arthur-Milchior** [comment]
Can you restart please

File: AnkiDroid/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -23,7 +23,8 @@
           package="com.ichi2.anki"
           android:installLocation="auto"
           android:versionCode="21000112"
-          android:versionName="2.10alpha12">
+          android:versionName="2.10alpha12"
```
Comment (mikehardy) [inline AnkiDroid/src/main/AndroidManifest.xml]: I looked at the release machinery to make sure this wouldn't mess it up, I don't think there is any dependency on where the closing '>' is, so this should be fine https://github.com/ankidroid/Anki-Android/blob/master/tools/release.sh#L53

**Comment by mikehardy** [comment]
> Can you restart please

yeah - that was a strange failure. emulator started in a way that our "wait for emulator" script thought it was up, then the emulator performed no useful work until timeout. First time I've seen that failure mode. Usually it's emulator fails startup, testAnswerCard failure, or testDebugUnitTest hang. 3 failure modes to triage was enough but I'll start watching/investigating this one as well

**Comment by mikehardy** [comment]
@Arthur-Milchior the conflict needs to be resolved but is just as a result of a version bump. @timrae I approved this but it needs your review and if it is to be merged it should go before next release or it will just conflict again...

**Comment by Arthur-Milchior** [comment]
Done

**Comment by mikehardy** [comment]
@Arthur-Milchior - backlog = backlog - 1 ;-) - thanks!

**Pull request merged**